src/Runner/Runner.php

```diff
@@ -168,14 +168,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                 }
             }
         } catch (\Exception $e) {
-            $this->dispatchEvent(
-                FixerFileProcessedEvent::NAME,
-                FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_EXCEPTION)
-            );
-
-            $this->errorsManager->report(new Error(Error::TYPE_EXCEPTION, $name));
-
-            return;
+            return $this->processException($name);
+        } catch (\Throwable $e) {
```

Comment:
I know PHP won't crash on it, but still it's ugly. Want to play with PHP7 features? use polyfill